scylla/src/transport/session.rs

```diff
@@ -285,6 +286,9 @@ pub struct SessionConfig {
     /// for e.g: if they do not want unexpected traffic
     /// or they expect the topology to change frequently.
     pub cluster_metadata_refresh_interval: Duration,
+
+    /// TODO(michael): Docs ...
+    pub replication_configs: ReplicationConfigs,
```

Comment:
`SessionBuilder` needs the corresponding setters as well, as it's the intended way to construct a `Session`.